### PR TITLE
Fixes #21258: Remove pagination on discovery page

### PR DESCRIPTION
**Pull request by danseethaler**
This removes extra blank rows when the discover page is first
loaded and also removes the pagination that isn't valid.

http://projects.theforeman.org/issues/21258

### CODE REVIEW

**Comment by danseethaler** [comment]
[test katello]

**Comment by waldenraines** [comment]
[test]

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/products/products.controller.js [hunk 1]
```diff
@@ -95,6 +95,14 @@ angular.module('Bastion.products').controller('ProductsController',
             ProductBulkAction.syncProducts(getBulkParams(), success, bulkError);
         };
 
+        $scope.goToDiscoveries = function (){
+            nutupane.table.rows = [];
+            nutupane.table.resource.results = [];
+            nutupane.table.resource.total = 0;
+            nutupane.table.resource.subtotal = 0;
```
Comment (waldenraines) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/products/products.controller.js]: Does simply calling `nutupane.invalidate()` not work here?

Comment (danseethaler) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/products/products.controller.js]: No, that calls the TableCache.removeTable() which blows away the whole table. I could probably remove the subtotal (line 102) but that wouldn't really help clean it up.

Comment (waldenraines) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/products/products.controller.js]: Oh I see.

**Comment by waldenraines** [comment]
Looks like these failures are legit:

```
19:18:19 app/assets/javascripts/bastion_katello/products/discovery/discovery.controller.js
19:18:19   141:38  error  Identifier 'per_page' is not in camel case  camelcase
19:18:19 
19:18:19 app/assets/javascripts/bastion_katello/products/products.controller.js
19:18:19   98:44  error  Missing space before opening brace  space-before-blocks
19:18:19 
19:18:19 ✖ 2 problems (2 errors, 0 warnings)
```

**Comment by danseethaler** [comment]
@waldenraines any thoughts on fixing that camel case error? Comment it out? Reference it some other way?

**Comment by waldenraines** [comment]
> @waldenraines any thoughts on fixing that camel case error? Comment it out? Reference it some other way?

Yeah, we usually just do `table['per_page']`.  Kind of silly but it works 😄

**Pull request merged**